platform/avr-raven/apps/raven-lcd-interface/raven-lcd.c

```diff
@@ -146,6 +146,9 @@ raven_ping6(void)
     UIP_ICMP_BUF->icmpchksum = 0;
     UIP_ICMP_BUF->icmpchksum = ~uip_icmp6chksum();
 
+#if UIP_CONF_IPV6_RPL
+    rpl_insert_header();
+#endif /* UIP_CONF_IPV6_RPL */
 
     tcpip_ipv6_output();
```

Comment:
Yes, but I guess we should try to do that to make the code look nicer! It might also be the case that we need a few checks before the call - but then it might make sense to have them in the function to avoid cluttering the caller(s) code.